lib/index.js

```diff
@@ -1,11 +1,43 @@
 const Rollbar = require('rollbar');
 
+const validLevels = [
+    'debug',
+    'info',
+    'warning',
+    'error',
+    'critical'
+];
+
 exports.register = (server, options) => {
     const rollbarClient = options.rollbarClient || new Rollbar(options);
     rollbarClient.omittedResponseCodes = options.omittedResponseCodes || [];
 
+    // Add Rollbar client to server
     server.decorate('server', 'rollbar', rollbarClient);
 
+    const sendRollbarMessage = async function sendRollbarMessage({ level = 'error', message }) {
+        if (validLevels.indexOf(level) === -1) {
+            this.log(['rollbar', 'error'], `Invalid level provided: ${level}. Must be one of [${validLevels.join(', ')}]`);
+            return false;
+        }
+
+        // Inject Person Tracking data
+        this.rollbar_person = this.auth.credentials;
+
+        const cb = function cb(err) {
+            if (err) {
+                this.log(['rollbar', 'error'], `Error reporting to rollbar, ignoring: ${err}`);
```

Comment:
Not using an arrow function here does not ensure a locked scope for the `this` variable usage. Are you sure about the context the `cb` is run with in the `rollbarClient[level]`?

Reply:
Hmmm, good call. I am honestly not at this point. It's difficult to test as well. I am going to work out a test for this to confirm.

Reply:
@mrlannigan I have updated the code and tests to verify this is working as we desire. Thank you for calling out the context when in the `cb`.